lm_eval/tasks/ja/jaqket_v2.py

```diff
@@ -0,0 +1,244 @@
+"""
+JAQKET: JApanese Questions on Knowledge of EnTitie
+https://www.anlp.jp/proceedings/annual_meeting/2020/pdf_dir/P2-24.pdf
+
+
+Homepage: https://www.nlp.ecei.tohoku.ac.jp/projects/jaqket/
+"""
+import os
+import inspect
+import datasets
+from math import exp
+from lm_eval.base import rf, Task
+from functools import partial
+from lm_eval.jasquad import jasquad
+
+_CITATION = """
+@InProceedings{Kurihara_nlp2020,
+  author =  "鈴木正敏 and 鈴木潤 and 松田耕史 and ⻄田京介 and 井之上直也",
+  title =   "JAQKET: クイズを題材にした日本語 QA データセットの構築",
+  booktitle =   "言語処理学会第26回年次大会",
+  year =    "2020",
+  url = "https://www.anlp.jp/proceedings/annual_meeting/2020/pdf_dir/P2-24.pdf"
+  note= "in Japanese"
+"""
+
+_TOP_K_RETRIEVER = 10
```

Comment:
Can you rename this from `retriever` to `limit` or something? When I saw `retriever` I thought this was a function or something dynamic rather than just a constant parameter.

Reply:
thank you for pointing it out, and I have now renamed it.